features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/SyslogReceiverNioDisruptorImpl.java

```diff
@@ -1,426 +0,0 @@
-/*******************************************************************************
```

Comment:
Unlike the JavaNet implementation, there is currently no way of using this one on either Minion (no blueprint) or OpenNMS (the class name isn't configurable) without code changes. It would also require significant refactoring to work with the Sink API.

That said, I don't see the point in maintaining it currently.

If we want to integrate it properly at some point in the future, we can reference the code available in any of the 18.x tags.